scripts/test_config.py

```
@@ -0,0 +1,299 @@
+        analysis = {
+            "total_test_files": 0,
+            "total_test_functions": 0,
+            "test_markers": {},
+            "test_categories": {},
+            "coverage_data": {},
+        }
+        
+        # Count test files and functions
+        for test_file in self.test_dir.rglob("test_*.py"):
+            analysis["total_test_files"] += 1
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)